packages/integration-tests/src/tests/api/application.test.ts

```diff
@@ -46,6 +46,24 @@ describe('admin console application', () => {
     expect(updatedApplication.oidcClientMetadata.redirectUris).toEqual(newRedirectUris);
   });
 
+  it('should update application "admin" successfully', async () => {
+    const application = await createApplication(
+      'test-update-is-admin',
+      ApplicationType.MachineToMachine
+    );
+    await updateApplication(application.id, {
+      isAdmin: true,
+    });
```

Comment:
Should we add a case that if the app type is not M2M, then the update should fail?

Reply:
LOG-6140, it seems to be a small bug